custom_components/bluecon/translations/pl.json

```diff
@@ -0,0 +1,69 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Zaloguj się za pomocą konta Fermax Blue",
+        "data": {
+          "username": "Nazwa użytkownika",
+          "password": "Hasło",
+          "client_id": "Identyfikator klienta Fermax API",
+          "client_secret": "Tajny klucz klienta API firmy Fermax",
+          "api_key": "Klucz interfejsu API Google",
+          "senderId": "FCM Sender ID (Identyfikator nadawcy FCM)",
+          "appId": "FCM App ID (Identyfikator aplikacji FCM)",
+          "projectId": "Identyfikator projektu FCM",
+          "packageName": "Nazwa pakietu FCM"
+        }
+      },
+      "reconfigure": {
+        "title": "Sign-in with Fermax Blue account",
```

Comment:
Hi @markopolopb!

Thanks for your contribution. Can you please update this label to match the `data.username` one? 

Thanks!